### ISSUE TITLE
ASSET [Pause Menu]

### ISSUE THREAD

**Issue by itspebbleboy** [GDACollab/TacoCat#298]
https://drive.google.com/drive/folders/1KwgSKjMNwBNdlJdj8CGFj1Qh5kaEwiRu

Sliders
(also text for buttons needs to be implemented)

**Comment by skysfalling** [comment]
I'm missing the volume sliders for some reason .. :/ pull from master please and see if you can address the issue

**Comment by YazmynS** [comment]
Weird cuz I haven’t done anything yet. I’ll see what I can do.

**Comment by YazmynS** [comment]
Actually I’ll be at todays meet and we can look together so I don’t mess
anything up.

**Comment by YazmynS** [comment]
Yeah I said I couldn’t do it. So it was supposed to go to someone else. I
haven’t even touched it yet so idk what happened.
